l10n_it_fatturapa_in/models/attachment.py

```diff
@@ -164,7 +171,6 @@ def _compute_xml_data(self):
 
             att.update(dict(
                 invoices_date=' '.join(invoices_date),
-                is_self_invoice=is_self_invoice,
```

Comment:
Questa modifica non va in f9191ab3879d00838c253d296085fa6caf989ec4 ?

Reply:
Concordo che andrebbe lì, non ce l'ho messa perché non è un mio commit.
Il commit è di @TheMule71, se lui è d'accordo mi co-autorizzo e ce la sposto :smile:

Reply:
Vai pure.

Reply:
Grazie, ho fatto; ho aggiunto nel commit anche la correzione della riga troppo lunga.

Quello che rimaneva non era gran che (il ricalcolo dei campi modificati fingendo la modifica del attachment) e l'ho schiacciato in un altro commit.